GitCommands/Git/GitItemStatus.cs

```diff
@@ -21,9 +30,30 @@ public class GitItemStatus : IComparable<GitItemStatus>
         public bool IsConflict { get; set; }
         public bool IsAssumeUnchanged { get; set; }
         public bool IsSkipWorktree { get; set; }
-        public bool IsStaged { get; set; } = true;
         public bool IsSubmodule { get; set; }
         public string RenameCopyPercentage { get; set; }
+        private StagedStatus StagedUnsafe { get; set; } = StagedStatus.Unknown;
```

Comment:
Is there any reason why we can't make this a backing field (e.g. `_staged`)? 
I don't particularly like current name as I don't find it very informative or meaningful.

Reply:
Will remain the field. See also next.

The reason was that in some situations it may be OK if Staged is not set, if information is supplied with something other than git-diff or git-status and you only are interested in where the field is actually set, no exception should occur. But I removed such usages and could set the field as private.

Reply:
> Will remain the field. 

You lost me here :)